autowarefoundation/autoware.universe
# feat(default_ad_api): apply route_selector

## Description

Apply interface changes from https://github.com/autowarefoundation/autoware.universe/pull/6363.

## Related links

- Related to https://github.com/autowarefoundation/autoware.universe/issues/6319
- Depends on https://github.com/tier4/tier4_autoware_msgs/pull/111
- Depends on https://github.com/autowarefoundation/autoware.universe/pull/6363

## Tests performed

1. Merge the dependent PRs.
2. Launch planning simulator.
3. Check if the route can be set. 

https://evaluation.tier4.jp/evaluation/reports/7d8458ad-5132-5f28-b0b2-1c117bc1b7e6?project_id=prd_jt

## Notes for reviewers

None

## Interface changes

Same as https://github.com/autowarefoundation/autoware.universe/pull/6363.

## Effects on system behavior

The MRM route will no longer be reflected in AD API.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.


## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.


## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.


After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/
